CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ## [1.4.1]:
 * Update Native Auth logging levels for improved clarity and consistency. #2184
 * Increased macOS minimum version to 10.15 #2220
+* Added Native Auth feature for MacOS to interact with the Microsoft Entra ID services
```

Comment:
Similar as the [other PR](https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2220/files#r1697312113), can you move this to a new entry?